composer/algorithms/utils/augmentation_common.py

```diff
@@ -43,7 +43,7 @@ def image_as_type(image: _InputImgT, typ: Type[_OutputImgT]) -> _OutputImgT:
         raise TypeError(f'Only typ={{torch.Tensor, Image}} is supported; got {typ}')
 
     if typ is torch.Tensor:
-        return cast(_OutputImgT, torchvision.transforms.functional.to_tensor(image))  # PIL -> Tensor
+        return cast(_OutputImgT, torchvision.transforms.functional.to_tensor(image))  # type: ignore PIL -> Tensor
```

Comment:
whats the new type error?

Reply:
Before they had type Any, now it is actually typed. It's something annoying of how it accepts any nd array (eg numpy) but we might pass in tensors because the input type is more generic. It's just annoyingness with how we make the input type more generic... and it's not worth patching this part of codebase